pkg/ddl/ingest/backend_mgr.go

```diff
@@ -62,11 +59,6 @@ func newLitBackendCtxMgr(ctx context.Context, sctx sessionctx.Context, path stri
 	if err != nil {
 		logutil.BgLogger().Warn("ingest backfill may not be available", zap.String("category", "ddl-ingest"), zap.Error(err))
 	}
-	isRaftKV2, err := util.IsRaftKv2(ctx, sctx)
```

Comment:
Why before we set up this check？for kv row format compatible？

Reply:
This is used to check if multi-rocks is enabled.

Reply:
OK